package.json

```diff
@@ -14,8 +14,8 @@
     "eslint": "npm run lint",
     "lint": "eslint .",
     "servertest": "npm run build && better-npm-run servertest",
-    "start:disco": "better-npm-run start:disco",
-    "start:search": "better-npm-run start:search",
+    "start:disco": "npm run version-check && NODE_PATH='./:./src' NODE_APP_INSTANCE=disco node bin/server.js",
+    "start:search": "npm run version-check && NODE_PATH='./:./src' NODE_APP_INSTANCE=search node bin/server.js",
```

Comment:
I refactored this to have a `start` again so once the deployments are updated to use `NODE_APP_INSTANCE=disco npm start` we can remove the old `start:disco` commands.